__tests__/App.test.js

```diff
@@ -1,10 +1,42 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { shallow, mount, render } from 'enzyme';
```

Comment:
Is mount and render needed here?

Reply:
not at the moment, but maybe for future testing? i'll be sure to update on next PR!